metadata/metadata/controllers/submission/handlers.py

```diff
@@ -102,6 +102,27 @@ def handle_submitter_accepts_policy(data: dict, agents: dict) \
     return tuple()
 
 
+def handle_submitter_contact_verified(data: dict, agents: dict) \
```

Comment:
Added this to support the submitter_contact_verified field in the submission schema.